docs/smart_contract_sdk/build_deploy_contract.md

```diff
@@ -53,7 +53,10 @@ cargo build --target wasm32-unknown-unknown --release
 </details>
 
 ## **Optimizing WASM binary for code size (optional)**
-The generated WASM binary has a large file size, which leads to higher gas costs for deployment and contract calls. You can compress and optimize the WASM binary through our `optimize_tool` in bash script. 
+
+The generated WASM binary has a large file size, which leads to higher gas costs for deployment and contract calls. You can compress and optimize the WASM binary through our `optimize_tool` in bash script in Linux or terminal commands in Windows.
```

Comment:
noted. We are going to push a section which covers for scripts for Linux/macOS and Windows